Docs/developer_guide/mrml_overview.md

```diff
@@ -117,3 +117,84 @@ Another view of [VTK/MRML pipeline for the 2D slice views](https://www.slicer.or
 
 Notes: the MapToWindowLevelColors has no lookup table set, so it maps the scalar volume data to 0,255 with no "color" operation.  This is controlled by the Window/Level settings of the volume display node.  The MapToColors applies the current lookup table to go from 0-255 to full RGBA.
 
+### Layout
+
+A layout manager ([qSlicerLayoutManager][qSlicerLayoutManager-apidoc]) shows or hides layouts:
+
+* It instantiates, shows or hides relevant view widgets.
+
+* It is associated with a [vtkMRMLLayoutNode][vtkMRMLLayoutNode-apidoc] describing the current layout configuration and
+  ensuring it can be saved and restored.
+
+* It owns an instance of [vtkMRMLLayoutLogic][vtkMRMLLayoutLogic-apidoc] that controls the layout node and the view
+  nodes in a MRML scene.
+
+* Pre-defined layouts are described using XML and are registered in `vtkMRMLLayoutLogic::AddDefaultLayouts()`.
+
+* Developer may register additionial layout.
+
+[qSlicerLayoutManager-apidoc]: http://apidocs.slicer.org/master/classqSlicerLayoutManager.html
+[vtkMRMLLayoutNode-apidoc]: http://apidocs.slicer.org/master/classvtkMRMLLayoutNode.html
+[vtkMRMLLayoutLogic-apidoc]: http://apidocs.slicer.org/master/classvtkMRMLLayoutLogic.html
+
+#### Registering a custom layout
+
+_TBD_

```

Comment:
You can find some material in the script repository. Maybe you can add a link to that.